config/routes.rb

```diff
@@ -1,5 +1,14 @@
 FloodRiskEngine::Engine.routes.draw do
   resources :enrollments, only: [:new, :create] do
     resources :steps, only: [:show, :update], controller: "enrollments/steps"
+
+    member do
```

Comment:
Seems to be polluting the state engine with functionality that should belong in a different controller